packages/image_picker/lib/image_picker.dart

```diff
@@ -6,47 +6,31 @@ import 'dart:async';
 import 'dart:io';
 
 import 'package:flutter/services.dart';
+import 'package:meta/meta.dart';
```

Comment:
I was just trying to publish the new version and pub is complaining that package:meta is not a dependency declared in the pubspec.yaml. If you have a change, can you send a follow-up PR that ads the dependency and I'll then publish the new version.

Reply:
Weird, shouldn't it automatically come with Flutter? I've never had to define package:meta as a dependency in pubspec.yaml if I was already depending on Flutter.

Reply:
import flutter foundation for meta

Reply:
@Hixie Thanks!